editor/components/block-list/index.js

```diff
@@ -250,7 +249,6 @@ class BlockList extends Component {
 
 		return (
 			<BlockSelectionClearer>
-				{ !! blocks.length && <BlockInsertionPoint /> }
```

Comment:
I'm not seeing the relation of this change to the others.What's the reason for the removal of this line? (sorry, I'm probably missing something)

Reply:
No you're not, this was a left-over from another PR. It's useless because the insertion point should not be shown here (it's shown inside each block). I've lost the link to the comment suggesting removing this.

It's in this PR because I thought it was related at first but it was not but I just kept this removal since its useless.

Reply:
Ok thank you for the clarification 👍